templates/admin/index/index.html.twig

```diff
@@ -186,7 +186,9 @@
     "lib/ext-plugins/portlet/PortalDropZone.js",
     "lib/ext-plugins/portlet/Portlet.js",
     "lib/ext-plugins/portlet/PortalColumn.js",
-    "lib/ext-plugins/portlet/PortalPanel.js"
+    "lib/ext-plugins/portlet/PortalPanel.js",
+    "../build/admin/ace-builds/src-min-noconflict/ace.js",
+    "../build/admin/ace-builds/src-min-noconflict/ext-modelist.js"
```

Comment:
getting error in the console now 
```
Refused to create a worker from 'blob:http://localhost/b02787d9-5feb-4b84-82c7-f5389283edf8' because it violates the following Content Security Policy directive: "script-src 'self'`.....
```

Reply:
The warning is only there on chrome and doesn't have any affect on the working of editor.